src/python/grpcio/grpc/_cython/_cygrpc/aio/completion_queue.pyx.pxi

```diff
@@ -0,0 +1,97 @@
+# Copyright 2020 The gRPC Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+cdef gpr_timespec _GPR_INF_FUTURE = gpr_inf_future(GPR_CLOCK_REALTIME)
+
+
+def _handle_callback_wrapper(CallbackWrapper callback_wrapper, int success):
```

Comment:
If eventually, this PR has success [1] and we are able to operate by using only one CQ paradigm (the polling one) we could reformulate the interface for calling the callbacks, right now we are coupled to the `completion_queue_for_callback` and might not be the most optimal one in the future if only the CQ next is used. 

[1] https://github.com/grpc/grpc/pull/22259

Reply:
Yes. This will be changed, and probably needs a new design.